### PR TITLE
Added delay argument in microserver

### PR DESCRIPTION
**Pull request by uthiramohan**

### CODE REVIEW

File: tests/tools/microServer/uWServer.py [hunk 1]
```diff
@@ -620,6 +622,11 @@ def main():
                         default=SERVER_PORT,
                         help="Port to use")
 
+    parser.add_argument("--delay", "-dy",
+                        type=int,
```
Comment (zizhong) [inline tests/tools/microServer/uWServer.py]: python sleep() supports float values.

**Comment by SolidWallOfCode** [comment]
[approve ci]

**Comment by persiaAziz** [comment]
squash please :)

**Comment by persiaAziz** [comment]
[approve ci]

**Pull request merged**